design/0001-metallb-crd.md

```diff
@@ -0,0 +1,173 @@
+# MetalLB Custom Resource Controller 
```

Comment:
It would be good to see a proposed CRD as part of this design.

Reply:
u want to share the CRD definition or sample config , sample config is already shared in the testing section

Reply:
It might be good to propose an initial CRD (rather than a sample) here to get an early review as this is a key part of the design. It may save you rework at a later stage.